src/Esri.ArcGISRuntime.Toolkit/Esri.ArcGISRuntime.Toolkit/Esri.ArcGISRuntime.Toolkit.csproj

```diff
@@ -23,6 +23,8 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetFramework)' == '$(AndroidTargetFramework)'">
+    <TargetFrameworkIdentifier>MonoAndroid</TargetFrameworkIdentifier>
+    <TargetFrameworkVersion>v8.0</TargetFrameworkVersion>
```

Comment:
This should already be set by the MSBuild SDK Extras package.

Reply:
I get spurious build errors when I don't have this.  Sometimes it fails, sometimes it succeeds.  I'll remove it for now, but I need to look into it more.